pycloudlib/gce/instance.py

```diff
@@ -42,8 +42,9 @@ def __init__(
         """
         if project is None or zone is None:
             raise ValueError(
-                "kwargs 'project' and 'zone' are required. "
-                "Project: {}, Zone: {}".format(project, zone)
+                "kwargs 'project' and 'zone' are required. Project: {}, Zone: {}".format(
+                    project, zone
+                )
```

Comment:
```suggestion
                "kwargs 'project' and 'zone' are required. Project: {project}, Zone: {zone}"
```

Reply:
@blackboxsw hehe you forgot the `f`
```suggestion
                f"kwargs 'project' and 'zone' are required. Project: {project}, Zone: {zone}"
```